### ISSUE TITLE
An `can't modify frozen String (FrozenError)` error occurred

### ISSUE THREAD

**Issue by tsub** [masutaka/circleci-bundle-update-pr#84]
The following error occurred in `$ circleci-bundle-update-pr hoge example@example.com $CIRCLE_BRANCH` command with circleci-bundle-update-pr v1.15.0 and Ruby v2.5.3.

```
Traceback (most recent call last):
	4: from /usr/local/bundle/bin/circleci-bundle-update-pr:23:in `<main>'
	3: from /usr/local/bundle/bin/circleci-bundle-update-pr:23:in `load'
	2: from /usr/local/bundle/gems/circleci-bundle-update-pr-1.15.0/bin/circleci-bundle-update-pr:15:in `<top (required)>'
	1: from /usr/local/bundle/gems/circleci-bundle-update-pr-1.15.0/lib/circleci/bundle/update/pr.rb:37:in `create_if_needed'
/usr/local/bundle/gems/circleci-bundle-update-pr-1.15.0/lib/circleci/bundle/update/pr.rb:147:in `update_pull_request_body': can't modify frozen String (FrozenError)
Exited with code 1
```

You may have degraded it with the following commit: https://github.com/masutaka/circleci-bundle-update-pr/commit/4f1409c91ad096050e51655f500bbeafcd19d13f

**Comment by masutaka** [comment]
Sorry just yanked v1.15.0.
https://rubygems.org/gems/circleci-bundle-update-pr/versions/1.15.0

The cause is here...
https://github.com/masutaka/circleci-bundle-update-pr/blob/v1.15.0/lib/circleci/bundle/update/pr.rb#L147

**Comment by masutaka** [comment]
Just released [v1.16.0](https://github.com/masutaka/circleci-bundle-update-pr/releases/tag/v1.16.0). thx! :bow:
